.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# Enforce top level code-owners CodeOwners
+*  @cloud-java-team-teamsync
+# Override the CodeOwners for the required submodules
+spring-cloud-gcp-bigquery/**  @cloud-java-frameworks-contributors-bigquery @cloud-java-team-teamsync
```

Comment:
Are these errors expected, or should these entries be `@GoogleCloudPlatform/cloud-java-team-teamsync` and `@GoogleCloudPlatform/cloud-java-frameworks-contributors-bigquery`?

Reply:
+1

Reply:
this seems to be valid with a `GoogleCloudPlatform` prefix

Reply:
also, both the mentioned groups can review `spring-cloud-gcp-bigquery` to avoid any bottlenecks, this can be extended for other modules as and when needed.